src/schema/artwork/__tests__/artwork.test.js

```diff
@@ -904,9 +904,22 @@ describe("Artwork type", () => {
       })
     })
 
-    it("returns false if artwork price is a range with multiple editions.", () => {
+    it("returns false if artwork price is a range with single edition.", () => {
```

Comment:
It’s not returning `false`. The description could also maybe be updated to something like `it returns that an edition set with a single edition has no price range` to better describe the intent.

Reply:
Well, that copy is not the right one for this test, so that goes to show how the intent isn’t made entirely clear at the moment 😄

Reply:
I think there should be 2 test for the artwork with one edition set. 
Something like: `Returns true for artwork with one edition set that is set as range`
and `Returns false for artwork with one edition with exact price`
I guess there is a potential 3rd case where we could test that it `Returns false for artwork with one edition set and no price info`